lib/routes/eventbrite/events.ts

```diff
@@ -0,0 +1,152 @@
+import { parseDate } from '@/utils/parse-date';
+import { load } from 'cheerio';
+import type { Context } from 'hono';
+
+export const route: Route = {
+    path: '/:region/:eventType?/:includePromoted?',
+    categories: ['other'],
+    example: '/eventbrite/canada--toronto/all-events',
+    parameters: { eventType: 'category of events for filtering', region: 'Region or scope of events' },
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    radar: [
+        {
+            source: ['eventbrite.com/d/:region/:eventType', 'eventbrite.ca/d/:region/:eventType'],
+            target: '/:region/:eventType',
+        },
+    ],
+    name: 'Events',
+    maintainers: ['elibroftw'],
+    handler: async (ctx: Context): Promise<Data> => {
+        const params = ctx.req.param();
+        const { region, includePromoted = 'false' } = params;
+        let eventType = params.eventType || 'all-events';
+        if (eventType === 'events') {
+            eventType = 'all-events';
+        }
+        const link = `https://www.eventbrite.com/d/${region}/${eventType}/`;
+        const response = await ofetch(link, {
+            headers: {
+                'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:138.0) Gecko/20100101 Firefox/138.0',
+            },
+        });
+        const $ = load(response);
+        const eventsApiUrl = new URL('https://www.eventbrite.com/api/v3/destination/events');
+        // exclude promoted events
+        const includePromotedBool = includePromoted !== 'false';
+        const eventListEl = $('div.search-results-panel-content section>ul>li')
+            .toArray()
+            .filter((item) => includePromotedBool || !$(item).text().includes('Promoted'));
+        const eventIds = eventListEl.map((item) => $(item).find('a.event-card-link').first().attr('data-event-id'));
+        eventsApiUrl.searchParams.append('event_ids', eventIds.join(','));
+        eventsApiUrl.searchParams.append('page_size', eventIds.length.toString());
+        eventsApiUrl.searchParams.append('expand', 'image,primary_venue,ticket_availability,primary_organizer');
+        const eventsData: EventbriteEvent[] = (await ofetch(eventsApiUrl.href)).events;
+        const items = eventListEl.map((item, index): DataItem => {
+            const el = $(item);
+            const a = el.find('.event-card-details a.event-card-link').first();
+            const fallbackTitle = a.attr('aria-label') || a.text() || a.toString();
+
+            const eventData = eventsData[index];
+            if (eventData === undefined) {
+                const pElements = el.find('p');
+                return {
+                    title: `should be ${fallbackTitle}`,
+                    author: pElements.length > 1 ? `should be ${$(pElements[1]).text()}` : 'unknown author',
```

Comment:
Do not use `should be` or `unknown author`. Leave it `undefined` if you don't know.